src/components/SignalGeneratorComponent.cpp

```diff
@@ -105,8 +105,11 @@ hyro::Result SignalGeneratorComponent::start ()
 
 hyro::Result SignalGeneratorComponent::update()
 {
+  std::chrono::seconds secs = std::chrono::duration_cast<std::chrono::seconds>(hyro::TimeUtils::Now_TimeSinceEpoch());
+  long int seconds          = secs.count(); //now expressed in seconds since epoch
+
   Signal value;
-  value.timestamp = 1000;
+  value.timestamp = seconds;
```

Comment:
Why not `value.timestamp = hyro::TimeUtils::Now_TimeSinceEpoch().count();` ?

Reply:
Good! I haven't thought this way, I'll change it and push again.